charts/incubator/blocky/templates/_webui.tpl

```diff
@@ -0,0 +1,36 @@
+{{- define "blocky.frontend" -}}
+image: {{ .Values.imageWebUI.repository }}:{{ .Values.imageWebUI.tag }}
+imagePullPolicy: {{ .Values.imageWebUI.pullPolicy }}
+securityContext:
+  runAsUser: 0
+  runAsGroup: 0
+  readOnlyRootFilesystem: false
+  runAsNonRoot: false
```

Comment:
It's a shame this is needed, but I think it's needed because of the use of port 80 here?

Reply:
I don't remember exactly, but had permission denied errors.

I'll take a look if there is a way to change internal port and try rootless.

Reply:
There is not, i've already filed the appropiate issues upstream